### PR TITLE
Add Japanese Translation.

### PR DESCRIPTION
**Pull request by silverskyvicto**
Signed-off-by: Yuuki Ebihara <yuuki.ebihara813@gmail.com>

### CODE REVIEW

File: config.toml [hunk 1]
```diff
@@ -150,6 +150,42 @@ weight = 5
...
+weight = 2
+
+[[languages.ja.menus.main]]
+name = "チャート"
+url = "https://hub.helm.sh/"
+weight = 3
+
+[[languages.ja.menus.main]]
+name = "ブログ"
+url = "/blog"
```
Comment (mattfarina) [inline config.toml:178]: Can you please point this url to "https://helm.sh/blog". That will correct some of the errors.

File: i18n/ja.toml [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+Helm は、Kubernetes アプリケーションの管理を支援します。Helm チャートは、最も複雑な Kubernetes アプリケーションの定義、インストール、およびアップグレードを支援します。
+
+チャートは簡単に作成、バージョン管理、共有、公開できます。そのため、Helm の使用を開始し、コピーアンドペーストを停止してください。
+
+Helm は [CNCF](https://cncf.io) の卒業プロジェクトであり、[Helm コミュニティ](https://github.com/helm/community)によって管理されています。
+
+### Learn more:
+
+* [Helm アーキテクチャ](./docs/topics/architecture/)
+* [クイックスタートガイド](./docs/intro/quickstart/)
```
Comment (mattfarina) [inline i18n/ja.toml:57]: These pages don't yet exist in Japanese. That means they are dead links. Can you please start these pages so the linking works.

File: config.toml (hunk 1 above)
Comment (silverskyvicto) [reply config.toml:178]: OK

File: i18n/ja.toml (hunk 2 above)
Comment (silverskyvicto) [reply i18n/ja.toml:57]: OK

**Comment by silverskyvicto** [comment]
@mattfarina 
The build seems to succeed by adding the original English page.

**Comment by flynnduism** [comment]
Hi @silverskyvicto thank you for updating this. I see the build is ✅  now. 

However, I am concerned about the copying of `en` docs into the `ja` site in order to avoid link issues and broken builds. It's a temporary way to avoid the broken links, but I don't think it's the best way to do this.

* could cause confusion between different languages  
  e.g.
  `https://deploy-preview-719--helm-merge.netlify.app/docs/chart_best_practices/pods/` and  
  `https://deploy-preview-719--helm-merge.netlify.app/ja/docs/chart_best_practices/pods/` are the same content but one is meant to be japanese 
* likelihood of incorrect docs; if the original file is updated but the version you have copied is not changed
* makes it harder to see what pages exist (i.e. are translated) and what pages are pending translation  

---  

I will follow with a suggestion to resolve the build link errors without putting `en` files into the `ja` site.

File: config.toml [hunk 3]
```diff
@@ -150,6 +150,42 @@ weight = 5
 [languages.ko.params]
 language_alternatives = ["en"]
 
+# Japanese
+[languages.ja]
+title = "Helm"
+description = "Helm - Kubernetes パッケージマネージャー"
+contentDir = "content/ja"
+languageName = "日本語"
```
Comment (flynnduism) [inline config.toml:158]: Can we update `languageName` to be  `日本語 Japanese` to include a translation. Preview:

Screen Shot 2020-06-24 at 10 20 00 AM

**Comment by flynnduism** [comment]
build error log:

```
8:52:30 AM: ========================================================================
8:52:30 AM: ja/docs/chart_template_guide/named_templates/index.html
8:52:30 AM:   target does not exist --- ja/docs/chart_template_guide/named_templates/index.html --> ../control_structures
8:52:30 AM: ja/docs/index.html
8:52:30 AM:   target does not exist --- ja/docs/index.html --> community
8:52:30 AM: ========================================================================
```

To fix the link issues, I suggest changes as follows:

1. Remove the community quicklink [here](https://github.com/silverskyvicto/helm-www/blame/add-translate-ja/content/ja/docs/_index.md#L21L23). this link is a shortcut, it is not critical
2. Change [this link](https://github.com/silverskyvicto/helm-www/blame/add-translate-ja/content/ja/docs/chart_template_guide/named_templates.md#L13) to an absolute url to point [here](https://helm.sh/docs/chart_template_guide/control_structures/)

**Comment by silverskyvicto** [comment]
Deleted pages that have not been translated into Japanese at all.

How about this?

**Comment by silverskyvicto** [comment]
Hi @mattfarina 

How about this pull request?
I would be happy if you could confirm.

**Comment by silverskyvicto** [comment]
Hi.

What about the situation?

**Comment by bridgetkromhout** [comment]
I checked with @mattfarina and as the tests now pass, we are happy to merge this and keep iterating on it. Thank you, @silverskyvicto!

**Pull request merged**